docs/features/techdocs/FAQ.md

```diff
@@ -57,3 +57,7 @@ your `mkdocs.yml` files per
 If the host name of your source code hosting URL does not include `github` or
 `gitlab`, an `integrations` entry in your `app-config.yaml` pointed at your
 source code provider is also needed (only the `host` key is necessary).
+
+#### What happens when you navigate to a TechDocs URL for an entity uses the `backstage.io/techdocs-entity` annotation?
```

Comment:
Can we move this to be right under the question about the `backstage.io/techdocs-ref` annotation, please?